File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/AssetReeferResponseReeferStatsFuelPercentage.scala

package org.openapitools.client.model


case class AssetReeferResponseReeferStatsFuelPercentage (
    /* Fuel percentage of the reefer. */
    _fuelPercentage: Option[Long],
    /* Timestamp in Unix milliseconds since epoch. */
    _changedAtMs: Option[Long]
)
object AssetReeferResponseReeferStatsFuelPercentage {
    def toStringBody(var_fuelPercentage: Object, var_changedAtMs: Object) =
        s"""
        | {
        | "fuelPercentage":$var_fuelPercentage,"changedAtMs":$var_changedAtMs
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/AssetReeferResponseReeferStatsFuelPercentage.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param fuelPercentage Fuel percentage of the reefer.
 * @param changedAtMs Timestamp in Unix milliseconds since epoch.
 */
data class AssetReeferResponseReeferStatsFuelPercentage (

        @JsonProperty("fuelPercentage") val fuelPercentage: Long? = null,

        @JsonProperty("changedAtMs") val changedAtMs: Long? = null
) {

}

